openedx/adg/lms/applications/test/test_views.py

```diff
@@ -0,0 +1,44 @@
+"""
+All tests for applications views
+"""
+from django.contrib.auth import get_user_model
+from django.test import Client, TestCase
+from django.test.client import RequestFactory
+
+from openedx.adg.lms.applications.views import ContactInformationView
+from openedx.adg.lms.registration_extension.models import ExtendedUserProfile
+from student.models import UserProfile
+
+
+class ContactInformationViewTest(TestCase):
+    """
+    Test cases for the ContactInformationView
+    """
+
+    def setUp(self):
+        """
+        Create a new user and logged in to access the view
+        """
+        self.client = Client()
+        self.user = get_user_model().objects.create_user(
+            email='testuser@test.com',
+            password='password',
+            username='testuser'
+        )
+        UserProfile.objects.create(user=self.user, name='testuser', city='XYZ')
+        ExtendedUserProfile.objects.create(user=self.user, saudi_national=False)
```

Comment:
Kindly, use a Factory for any object creation as mentioned above. If a factory does not exist, let's please create one.